daemon/cluster/services.go

```diff
@@ -139,9 +139,16 @@ func (c *Cluster) CreateService(s types.ServiceSpec, encodedAuth string, queryRe
 		case *swarmapi.TaskSpec_Generic:
 			switch serviceSpec.Task.GetGeneric().Kind {
 			case string(types.RuntimePlugin):
+				info, _ := c.config.Backend.SystemInfo()
+				if !info.ExperimentalBuild {
+					return fmt.Errorf("runtime type %q only supported in experimental", types.RuntimePlugin)
```

Comment:
I agree that `runtime` would make me think of `runc` / OCI runtime. More so because there's ac`runtimes` configuration on the daemon.

"Swarm task type" or "payload" perhaps looks better